core/web/resolver/mutation.go

```diff
@@ -835,7 +835,7 @@ func (r *Resolver) CreateChain(ctx context.Context, args struct {
 		chainCfg.KeySpecific = sCfgs
 	}
 
-	chain, err := r.App.GetChainSet().Add(id.ToInt(), *chainCfg)
+	chain, err := r.App.GetChains().EVM.Add(id.ToInt(), *chainCfg)
```

Comment:
Is the GQL schema scoped appropriately?

Reply:
I haven't made changes to GQL or any UI yet https://app.shortcut.com/chainlinklabs/story/26276/multi-chain-type-ui-node-chain-configuration